Index: app/release/output-metadata.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"version\": 2,\r\n  \"artifactType\": {\r\n    \"type\": \"APK\",\r\n    \"kind\": \"Directory\"\r\n  },\r\n  \"applicationId\": \"com.nikealarm.nikedrawalarm\",\r\n  \"variantName\": \"processReleaseResources\",\r\n  \"elements\": [\r\n    {\r\n      \"type\": \"SINGLE\",\r\n      \"filters\": [],\r\n      \"versionCode\": 16,\r\n      \"versionName\": \"2.2.3\",\r\n      \"outputFile\": \"app-release.apk\"\r\n    }\r\n  ]\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/release/output-metadata.json	(revision 6a841d44a99e3adf8afa354edc7b75fd8080a9a7)
+++ app/release/output-metadata.json	(date 1613971486993)
@@ -10,8 +10,8 @@
     {
       "type": "SINGLE",
       "filters": [],
-      "versionCode": 16,
-      "versionName": "2.2.3",
+      "versionCode": 17,
+      "versionName": "2.2.4",
       "outputFile": "app-release.apk"
     }
   ]
diff --git .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-22__1_17__Default_Changelist_.xml .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-22__1_17__Default_Changelist_.xml
diff --git .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-22__2_01__Default_Changelist_.xml .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-22__2_01__Default_Changelist_.xml
diff --git .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-22__1_52__Default_Changelist_.xml .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-22__1_52__Default_Changelist_.xml
diff --git .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-21__12_48__Default_Changelist_.xml .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-21__12_48__Default_Changelist_.xml
diff --git .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-20__11_47__Default_Changelist_1.xml .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-20__11_47__Default_Changelist_1.xml
diff --git .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-20__11_47__Default_Changelist_.xml .idea/shelf/Uncommitted_changes_before_Checkout_at_2021-02-20__11_47__Default_Changelist_.xml
